VERSION

```diff
@@ -1 +1 @@
-v0.7.0
+v0.8.0
```

Comment:
I bumped up the version since some events will be dispatched to new methods of  IEventService.